plugin/type_mysql_json/type.cc

```diff
@@ -62,7 +62,7 @@ class Field_mysql_json: public Field_blob
   bool parse_mysql(String *dest, const char *data, size_t length) const;
   bool send(Protocol *protocol) { return Field::send(protocol); }
   void sql_type(String &s) const
-  { s.set_ascii(STRING_WITH_LEN("json /* MySQL 5.7 */")); }
+  { s.set_ascii(STRING_WITH_LEN("longtext /* MySQL 5.7 */")); }
```

Comment:
Better `/* JSON from MySQL 5.7 */`
because `longtext` is the same in MariaDB and in MySQL 5.7, so it's unclear why `SHOW CREATE` will print this comment.

Reply:
Thanks, have updated.